apps/src/music/progress/MusicValidator.ts

```diff
@@ -1,16 +1,17 @@
 // Music Lab specific validations.
 
 import MusicPlayer from '../player/MusicPlayer';
+import {Condition} from '@cdo/apps/lab2/types';
 import ConditionsChecker, {
-  KnownConditions,
+  ConditionNames,
 } from '@cdo/apps/lab2/progress/ConditionsChecker';
 import {PlaybackEvent} from '../player/interfaces/PlaybackEvent';
 import {Validator} from '@cdo/apps/lab2/progress/ProgressManager';
 
-const KnownConditions: KnownConditions = {
-  PLAYED_ONE_SOUND: 'played_one_sound',
-  PLAYED_TWO_SOUNDS_TOGETHER: 'played_two_sounds_together',
-  PLAYED_THREE_SOUNDS_TOGETHER: 'played_three_sounds_together',
+export const ConditionNamesList: ConditionNames = {
+  PLAYED_SOUNDS_TOGETHER: 'played_sounds_together',
+  PLAYED_SOUND_TRIGGERED: 'played_sound_triggered',
+  USED_BLOCK: 'used_block',
```

Comment:
Are we still using `"USED_BLOCK"`? Will we need to support this in the future?

Reply:
Oh, will remove for now.